content/download.html

```diff
@@ -122,10 +122,10 @@ <h3><img src="/images/platforms/arch-small.svg" class="downloadsmallplatform" />
   <p>Stable releases are available in the <a href="https://www.archlinux.org/packages/community/any/openra/">official Arch repository</a>, and can be installed using <span style="font-family: monospace;">pacman</span>:<pre>$ pacman -S openra </pre></p>
   <p class="downloadthirdparty">We do not maintain this external package source, so there may be delays when a new version is released.<br />Please contact the downstream repository maintainers about any packaging issues.</p>
   <h3><img src="/images/platforms/gentoo-small.svg" class="downloadsmallplatform" />Gentoo</h3>
-  <p>Unofficial packages for both releases and playtests are available via the &ldquo;dr&rdquo; overlay:</p>
+  <p>Unofficial packages for both releases and playtests are available via the &ldquo;dr&rdquo; overlay. You only need the dotnet overlay if you compile openra with the nuget USE flag.</p>
```

Comment:
It looks like the ebuild has bitrotted and the nuget USE flag doesn't do anything at all: it attempts to patch a check that no longer exists - nuget is now never used on Linux.

I'd prefer it if we could drop that flag in the ebuild and then remove the `dotnet` overlay reference completely.

Reply:
Ping @cerebrum.

Reply:
https://github.com/cerebrum/dr/commit/3ae48584f360ac92d51e28b3bee36968cd9428b5 removed the `dotnet` requirement from the ebuild, so this PR can now be updated.